translations/messages+intl-icu.de.yml

```diff
@@ -889,7 +889,7 @@ element.assigned: "Bezug zu Planungsdokument"
 element.delete: "Element entfernen"
 element.notassigned: "Es ist kein Bezug zu einem Planungsdokument festgelegt."
 element.selected: "Ausgewähltes Element"
-element.state.autoswitch: "Status der Kategorie automatisch auf \"{state}\" umstellen"
+element.state.autoswitch: "Status der Kategorie(n) automatisch auf \"{state}\" umstellen"
```

Comment:
See above comment. I'd revert the change to `element.state.autoswitch` and add a new key instead.